lightning/src/util/enforcing_trait_impls.rs

```diff
@@ -46,37 +45,39 @@ pub const INITIAL_REVOKED_COMMITMENT_NUMBER: u64 = 1 << 48;
 #[derive(Clone)]
 pub struct EnforcingSigner {
```

Comment:
Does the list of policy checks above need updating?

Reply:
Updated